library/src/main/java/net/alhazmy13/hijridatepicker/date/hijri/HijriDatePickerDialog.java

```diff
@@ -948,9 +948,9 @@ private void adjustDayInMonthIfNeeded(UmmalquraCalendar calendar) {
     @Override
     public void onClick(View v) {
         tryVibrate();
-        if (v.getId() == R.id.mdtp_date_picker_year) {
+        if (v.getId() == R.id.mdtp_hijri_date_picker_year) {
             setCurrentView(YEAR_VIEW);
-        } else if (v.getId() == R.id.mdtp_date_picker_month_and_day) {
+        } else if (v.getId() == mdtp_hijri_date_picker_month_and_day) {
```

Comment:
It should be `R.id.mdtp_hijri_date_picker_month_and_day`, please fix it to approve your PR .